test.js

```diff
@@ -23,6 +23,19 @@ it('should trim off \\r\\n', function () {
 it('should not be susceptible to exponential backtracking', function () {
 	var start = Date.now();
 	trimOffNewlines('a' + '\r\n'.repeat(1000) + 'a');
+	trimOffNewlines(`a${'\r\n'.repeat(1000)}a`.repeat(1000));
```

Comment:
This additional test fails with the current code but succeeds with the change here.

Reply:
Because of the template string, the test won't run in Node.js 0.10, which the `package.json` says is supported.

Reply:
I've updated the test to work in Node.js 0.10.x but `mocha` and `xo` are still incompatible.